tests/unittest/_torch/attention/sparse/dsa/test_dsa_indexer.py

```
@@ -2606,6 +2606,136 @@ def test_indexer_decode_custom_vs_fallback(batch_size, next_n, index_topk, seq_l
+    )
+    meta0.radix_aux_logits = torch.zeros(
+        (_radix_gen, _radix_bp, index_topk), device="cuda", dtype=torch.float32
+    )
+
+    meta0.in_mtp_draft_loop = True
+
+    h, q, k_fp8, k_scale, w = make_inputs(step0_tokens)
+    indexer._update_k_cache(k_fp8, k_scale, meta0)
+    try:
+        topk0 = indexer.sparse_attn_indexer(meta0, h, q, k_fp8, k_scale, w, use_custom_topk=True)
+    except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)